### PR TITLE
Changes required to release Core

### PR DESCRIPTION
**Pull request by wtrocki**
### Motivation

Adjust repository structure in order to support release.
Test release using different name.
We are making assumption that all packages will use matching versions.

If we can approve current aproach I will merge that and do proper release of the core. 

### Progress

- [x] Publish testing pod
https://github.com/CocoaPods/Specs/commit/f91fee0a68cbe8ff0ffe109733f068f23a1f285a
- [x] Use test pod in example project
```
Downloading dependencies
Installing Alamofire (4.7.2)
Installing ObjcExceptionBridging (1.0.1)
Installing TestCorePublish (0.1.0)
Installing XCGLogger (6.0.2)
```

screen shot 2018-04-17 at 4 44 37 pm

## Separate PR
- [ ] Publish core after merge
- [ ] Document the whole process.

### CODE REVIEW

File: example/Podfile [hunk 1]
```diff
@@ -19,5 +19,22 @@ target 'AeroGearSdkExample' do
...
+# end
+
+# target 'AeroGearSdkAuth' do
+#   use_frameworks!
+#   pod 'AGSAuth', :path => '../modules/auth'
+
+#   target 'AeroGearSdkAuthTests' do
+#     inherit! :search_paths
+#   end
+# end
```
Comment (wei-lee) [inline example/Podfile]: should the commented code be removed?

File: AGSCore.podspec [hunk 2]
```diff
@@ -10,9 +10,9 @@ Pod::Spec.new do |s|
   s.authors      = 'AeroGear'
   s.platform     = :ios, '9.0'
   s.ios.deployment_target = '9.0'
-  s.source       = { :git => 'https://github.com/aerogear/aerogear-ios-sdk.git',
-                     :tag => s.version}
-  s.source_files = '**/*.swift'
+  s.source       = { :git => 'https://github.com/wtrocki/aerogear-ios-sdk.git',
```
Comment (wei-lee) [inline AGSCore.podspec]: you might want to change the url here to aerogear

**Comment by wtrocki** [comment]
@wei-lee I have spent very long time trying to do subspec validation to work. Mostly dealing with some cocoapods issues. Decided to use current aproach as it took just to long.

@wei-lee We could configure travis to do releases when we create new release on github. WDYT?

**Comment by wei-lee** [comment]
@wtrocki that will be great. But I don't think it's required as part of this PR. We can add automation later.

**Pull request merged**